templates/mfa/index.html

```diff
@@ -0,0 +1,69 @@
+{% extends "mfa/base_manage.html" %}
+
+{# https://github.com/pennersr/django-allauth/blob/main/allauth/templates/mfa/index.html #}
+
+
+{% load allauth %}
+{% load i18n %}
+
+
+{% block content %}
+
+  <h1>{% block heading %}{% trans "MFA" %}{% endblock heading %}</h1>
+
+  <p>{% trans 'Configure or update your multi-factor authentication settings.' %}</p>
+
+
+  {% if "totp" in MFA_SUPPORTED_TYPES %}
+
+  <section class="my-5" id="mfa-totp">
+    <h3>{% trans "Authenticator App" %}</h3>
+
+    {% url 'mfa_deactivate_totp' as deactivate_url %}
+    {% url 'mfa_activate_totp' as activate_url %}
+
+    {% if authenticators.totp %}
+      <p>
+        {% blocktrans trimmed %}
+          Two-factor authentication using an authenticator app is enabled.
```

Comment:
Are we calling it 2FA or MFA? We should try to use the same language.

Reply:
Sounds good. I'll make sure this fits. If it doesn't, I'll just be consistent in what we're calling it.